### PR TITLE
[7.0] [ADD] account invoice default salesteam

### PR DESCRIPTION
**Pull request by veloutin**
This module set the Sales Team (section_id field) in the 2 following scenarios:
- when creating an invoice, set on partner_id.onchange() from the partner's
  Sales Team
- when invoicing from sales_order, set from the Sales Order's Sales Team.

### CODE REVIEW

**Comment by max3903** [comment]
:+1:

File: account_invoice_default_salesteam/account_invoice.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+###############################################################################
+
+from openerp.osv import orm
+
+
+class AccountInvoice(orm.Model):
+    _inherit = "account.invoice"
+
+    def create(self, cr, uid, values, context=None):
+        if "partner_id" in values and not values.get("section_id"):
```
Comment (pedrobaeza) [inline account_invoice_default_salesteam/account_invoice.py:30]: Why do you need this patch at creation time?

Comment (veloutin) [inline account_invoice_default_salesteam/account_invoice.py:30]: To use the partner's sales team if we did not supply one.

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**